src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -760,6 +765,23 @@ private MenuBar createMenu() {
         );
 
         final PushToApplicationAction pushToApplicationAction = new PushToApplicationAction(stateManager, this.getPushApplications(), this.getDialogService());
+        MenuItem pushToApplicationMenuItem = factory.createMenuItem(pushToApplicationAction.getActionInformation(), pushToApplicationAction);
+
+        final Menu pushApplicationsMenu = factory.createSubMenu(StandardActions.PUSH_APPLICATION);
+        final ToggleGroup pushToToggleGroup = new ToggleGroup();
+        RadioMenuItem pushApplication;
+        PushToApplicationMenuAction pushToApplicationMenuAction;
+
+        for (PushToApplication application : JabRefFrame.this.getPushApplications().getApplications()) {
```

Comment:
this.getPushApplications?

Reply:
Thanks, I copied this for-loop out of externalTab and did not simplify it yet.